k8s/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright The Kubernetes Authors.
```

Comment:
Should change to Longhorn?

Reply:
Copied from other k8s codes.
Does Longhorn have template for the copyright infromation?

Reply:
How about
```
Copyright (c) The Longhorn Authors

Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
```

Reply:
Updated. Thank you.